src/common/categories-results.php

```diff
@@ -0,0 +1,201 @@
+<?php
+  session_start();
+
+  if(!isset($_SESSION['user_id'])) {
+    header('Location: /tindahan.ph/src/common/login.php?mode=login');
+    exit();
+  }
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>tindahan.ph - Categories</title>
+
+    <link
+      rel="icon"
+      type="image/png"
+      href="../../assets/images/tph-logo-128px.png"
+    />
+
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"
+      integrity="sha512-894YE6QWD5I59HgZOGReFYm4dnWc1Qt5NtvYSaNcOP+u1T9qYdvdihz0PPSiiqn/+/3e7Jo4EaG7TubfWGUrMQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    ></script>
+    <script
+      src="https://kit.fontawesome.com/056f419e6a.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <link rel="stylesheet" href="../../css/base/base.css" />
+    <link rel="stylesheet" href="../../css/components/components.css" />
+    <link rel="stylesheet" href="../../css/utilities/utilities.css" />
+    <link rel="stylesheet" href="../../css/common/common.css" />
+
+    <script src="../../js/common/auto-resizer.js"></script>
+    <script src="../../js/common/auth/logout.js"></script>
+    <script src="../../js/common/categories-results.js"></script>
+  </head>
+
+  <script>
+    new Promise(function (resolve, reject) {
```

Comment:
For security purposes, we will have no in-document JavaScript. Please move this to this page's controller/js file.